Release_Notes.md

```diff
@@ -0,0 +1,7 @@
+# vNext
```

Comment:
If you don't feel strongly about the file name here, we've been sticking to this spec:
https://keepachangelog.com/en/1.0.0/

Which would rename this file to be `CHANGELOG.md`.

And yes, I acknowledge the irony of the fact that you literally just renamed this file. I am rarely accused of having really great timing.

Reply:
I'm fine with whatever team standards are